src/styles/layouts/_layout-account.scss

```diff
@@ -0,0 +1,65 @@
+.layout-account {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: rem(40px) repeat(2, 1fr) rem(200px);
+  grid-template-areas:
+    'header'
+    'sidebar'
+    'main'
+    'footer';
+
+  height: 100vh;
+
+  @include respond-above('md') {
+    grid-template-rows: rem(80px) repeat(2, 1fr) rem(250px);
+  }
+
+  @include respond-above('lg') {
+    grid-template-columns: minmax(350px, 450px) minmax(642px, 1fr);
+    grid-template-rows: rem(100px) 1fr rem(300px);
+    grid-template-areas:
+      'header header'
+      'sidebar main'
+      'footer footer';
+  }
+
+  @include respond-above('xl') {
+    grid-template-rows: rem(140px) 1fr rem(350px);
+  }
+
+  & > header {
+    grid-area: header;
+  }
+
+  & > aside {
```

Comment:
Currently you are setting the height at the `account-information`, how about using `padding` here and set `height: 100%` at `account-information` so it's even more responsive (imagine a user have a long/short bio)